boards/arm/lpcxpresso55s69/doc/index.rst

```diff
@@ -92,7 +92,9 @@ CPU0 is the only target that can run standalone.
 - *lpcxpresso55s69_cpu1* CPU1 target, no security extensions
 
 NS target for CPU0 does not work correctly without a secure image enabling it.
-To enable it, run any of the ``tfm_integration`` samples.
+To enable it, run any of the ``tfm_integration`` samples. When using the NS
+target ``CONFIG_BUILD_WITH_TFM`` is enabled to ensure that a valid S image is
+included during the build process.
 
 CPU1 does not work without CPU0 enabling it.
 To enable it, run one of the following samples in ``subsys\ipc``:
```

Comment:
The documentation states we should flash tfm_merged.bin. I see only a .hex file, should this be updated to tfm_merged.hex